tests/end-to-end/simple/App.java

```diff
@@ -0,0 +1,185 @@
+package com.acme.test.simple;
+
+
+import software.constructs.Construct;
+
+import java.util.*;
+import java.util.stream.Collectors;
+import software.amazon.awscdk.*;
+import software.amazon.awscdk.App;
+import software.amazon.awscdk.CfnMapping;
+import software.amazon.awscdk.CfnTag;
+import software.amazon.awscdk.Stack;
+import software.amazon.awscdk.StackProps;
+
+
+import software.amazon.awscdk.services.s3.*;
+import software.amazon.awscdk.services.sqs.*;
+
+public class NoctApp {
+  public static void main(final String[] args) {
+    App app = new App();
+    StackProps props = StackProps.builder()
+      .description("An example stack that uses many of the syntax elements permitted in a" + 
+       "CloudFormation template, but does not attempt to represent a realistic stack.")
+    .build();
+    new NoctStack(app, "MyProjectStack", props);
+    app.synth();
+  }
+}
+interface NoctStackProps extends StackProps {
+}
+class NoctStack extends Stack {
+  public NoctStack(final Construct scope, final String id) {
+    super(scope, id, null);
+  }
+  public NoctStack(final Construct scope, final String id, final StackProps props) {
+    super(scope, id, props);
+    { // Start Mapping section
```

Comment:
If we wrap the mappings section in `{}` I guess none of the bindings it makes are visible in the rest of the code? Maybe should remove the braces there & optionally just keep a leading comment?